### ISSUE TITLE
TypeOrmHealthIndicator type issue

### ISSUE THREAD

**Issue by dfee** [nestjs/terminus#545]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
There is a type error on `TypeOrmHealthIndicator.pingCheck`. Providing an instance of `Connection` throws a type error

## Minimal reproduction of the problem with instructions
```ts
import { Injectable } from "@nestjs/common";
import {
  TerminusEndpoint,
  TerminusModuleOptions,
  TerminusOptionsFactory,
  TypeOrmHealthIndicator,
} from "@nestjs/terminus";
import { Connection } from "typeorm";

@Injectable()
export class TerminusOptionsService implements TerminusOptionsFactory {
  private readonly _myConn!: Connection;
  constructor(private readonly _typeOrm: TypeOrmHealthIndicator) {}

  createTerminusOptions(): TerminusModuleOptions {
    const connection: Connection = new Connection({
      type: "postgres",
      url: "psql://",
    });
    const healthEndpoint: TerminusEndpoint = {
      healthIndicators: [
        async () => this._typeOrm.pingCheck("database", { connection }),
      ],
      url: "/health",
    };
    return { endpoints: [healthEndpoint] };
  }
}
```

Produces the type error (on `connection` inside `TypeOrmPingCheckSettings`):
```
Argument of type '{ connection: Connection; }' is not assignable to parameter of type 'TypeOrmPingCheckSettings'.
  The types of 'connection.options' are incompatible between these types.
    Type 'import("/Users/dfee/myapp/node_modules/typeorm/connection/ConnectionOptions").ConnectionOptions' is not assignable to type 'import("/Users/dfee/myapp/node_modules/@nestjs/terminus/node_modules/typeorm/connection/ConnectionOptions").ConnectionOptions'.
      Type 'MysqlConnectionOptions' is not assignable to type 'ConnectionOptions'.
        Type 'import("/Users/dfee/myapp/node_modules/typeorm/driver/mysql/MysqlConnectionOptions").MysqlConnectionOptions' is not assignable to type 'import("/Users/dfee/myapp/node_modules/@nestjs/terminus/node_modules/typeorm/driver/mysql/MysqlConnectionOptions").MysqlConnectionOptions'.
          Types of property 'entities' are incompatible.
            Type '(string | Function | import("/Users/dfee/myapp/node_modules/typeorm/entity-schema/EntitySchema").EntitySchema<any>)[] | undefined' is not assignable to type '(string | Function | import("/Users/dfee/myapp/node_modules/@nestjs/terminus/node_modules/typeorm/entity-schema/EntitySchema").EntitySchema<any>)[] | undefined'.
              Type '(string | Function | import("/Users/dfee/myapp/node_modules/typeorm/entity-schema/EntitySchema").EntitySchema<any>)[]' is not assignable to type '(string | Function | import("/Users/dfee/myapp/node_modules/@nestjs/terminus/node_modules/typeorm/entity-schema/EntitySchema").EntitySchema<any>)[]'.
                Type 'string | Function | import("/Users/dfee/myapp/node_modules/typeorm/entity-schema/EntitySchema").EntitySchema<any>' is not assignable to type 'string | Function | import("/Users/dfee/myapp/node_modules/@nestjs/terminus/node_modules/typeorm/entity-schema/EntitySchema").EntitySchema<any>'.
                  Type 'EntitySchema<any>' is not assignable to type 'string | Function | EntitySchema<any>'.
                    Type 'import("/Users/dfee/myapp/node_modules/typeorm/entity-schema/EntitySchema").EntitySchema<any>' is not assignable to type 'import("/Users/dfee/myapp/node_modules/@nestjs/terminus/node_modules/typeorm/entity-schema/EntitySchema").EntitySchema<any>'.
                      The types of 'options.columns' are incompatible between these types.
                        Type '{ [x: string]: import("/Users/dfee/myapp/node_modules/typeorm/entity-schema/EntitySchemaColumnOptions").EntitySchemaColumnOptions | undefined; }' is not assignable to type '{ [x: string]: import("/Users/dfee/myapp/node_modules/@nestjs/terminus/node_modules/typeorm/entity-schema/EntitySchemaColumnOptions").EntitySchemaColumnOptions | undefined; }'.
                          Index signatures are incompatible.
                            Type 'import("/Users/dfee/myapp/node_modules/typeorm/entity-schema/EntitySchemaColumnOptions").EntitySchemaColumnOptions | undefined' is not assignable to type 'import("/Users/dfee/myapp/node_modules/@nestjs/terminus/node_modules/typeorm/entity-schema/EntitySchemaColumnOptions").EntitySchemaColumnOptions | undefined'.
                              Type 'import("/Users/dfee/myapp/node_modules/typeorm/entity-schema/EntitySchemaColumnOptions").EntitySchemaColumnOptions' is not assignable to type 'import("/Users/dfee/myapp/node_modules/@nestjs/terminus/node_modules/typeorm/entity-schema/EntitySchemaColumnOptions").EntitySchemaColumnOptions'.
                                Types of property 'type' are incompatible.
                                  Type 'import("/Users/dfee/myapp/node_modules/typeorm/driver/types/ColumnTypes").ColumnType' is not assignable to type 'import("/Users/dfee/myapp/node_modules/@nestjs/terminus/node_modules/typeorm/driver/types/ColumnTypes").ColumnType'.
                                    Type '"smalldecimal"' is not assignable to type 'ColumnType'.ts(2345)
```

## Environment

<pre><code>
Nest version: 6.11.5

 
For Tooling issues:
- Node version: 13.3.0
- Platform:  Mac

Others:
- typescript: 3.7.5
- typeorm: 0.2.22
- @nestjs/typeorm: 6.2.0

**Comment by BrunnerLivio** [comment]
It seems to be a typeorm version mismatch. Will publish fix shortly.

**Comment by BrunnerLivio** [comment]
Fixed in 6.5.6 :tada:
